<launch>
    <node pkg="libuvc_camera" type="camera_node" respawn="false" name="logitech_camera" output="screen"/>

    <arg name="gui" default="false"/>
    <arg name="image_topic" default="/image_raw"/>
    <arg name="objects_path" default="$(find solution_3)/objects"/>
    <arg name="settings_path" default="$(find solution_3)/objects/settings.ini"/>

	<node name="find_object_2d" pkg="find_object_2d" type="find_object_2d" output="screen">
		<remap from="image" to="$(arg image_topic)"/>
		<param name="gui" value="$(arg gui)" type="bool"/>
		<param name="objects_path" value="$(arg objects_path)" type="str"/>
		<param name="settings_path" value="$(arg settings_path)" type="str"/>
	</node>

    <node pkg="solution_3" type="detect_objects_node.py" name="detect_objects_node">
        <rosparam file="$(find solution_3)/launch/detect_objects.yaml" command="load"/>
        <param name="image_topic" value="$(arg image_topic)"/>
    </node>

</launch>